tools/validators/instance_validator/validate/handler.py

```diff
@@ -123,34 +141,49 @@ def RunValidation(filenames: List[str],
   """
   saved_stdout = sys.stdout
   report_file = None
+
+  print('[INFO]\tStarting validation process.')
   if report_filename:
     # pylint: disable=consider-using-with
     report_file = open(report_filename, 'w', encoding='utf-8')
     sys.stdout = report_file
   try:
-    print('\nStarting validator...\n')
-    print('\nStarting universe generation...\n')
+    print('[INFO]\tLoading ontology.')
     universe = generate_universe.BuildUniverse(
         use_simplified_universe=use_simplified_universe,
         modified_types_filepath=modified_types_filepath,
         default_types_filepath=default_types_filepath)
     if not universe:
-      print('\nError generating universe')
+      print('[ERROR]\tUniverse did not load properly.')
       sys.exit(0)
-    print('\nStarting config validation...\n')
+    print('[INFO]\tOntology loaded.')
+
     entities = _ValidateConfig(filenames, universe, is_udmi)
     if subscription:
-      print('\nStarting telemetry validation...\n')
+      print('[INFO]\tStarting telemetry validation.')
       _ValidateTelemetry(subscription, service_account, entities,
                          timeout, is_udmi)
     else:
-      print('Subscription is needed for telemetry validation')
-
+      print('[WARNING]\tTelemetry validation skipped, subscription '
+            'not found. Please provide a subscription and service account to '
+            'run telemetry validation. See here for more details: '
+            'https://google.github.io/digitalbuildings/tools/validators/'
+            'instance_validator/#telemetry-validation')
+  # pylint: disable=broad-except
+  # Using broad exception to catch any underlying error that might cause
+  # validation to fail
+  except Exception as ex:
```

Comment:
this is not the best practice here. any reason why not using a specific exception?

Reply:
if the exception is already thrown before, there is no need to catch it here again? we can remove this block?
Or leave a general message?
    print(f'[ERROR]\t Validation failed See logs above and error here: {ex}.')